inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/network/ClientMgr.java

```diff
@@ -650,6 +666,25 @@ private void fillUpWorkClientWithLastBadClient() {
         }
     }
 
+    private void fillUpWorkClientWithBadClient(List<HostInfo> badHostLists) {
+        if (badHostLists.isEmpty()) {
```

Comment:
It is good thing that badHostLists is empty, why use warn level log?

Reply:
> It is good thing that badHostLists is empty, why use warn level log?

Invoking this method means that all hosts are bad, so badHostLists should not be empty